doc/MSTP/MSTP.md

```diff
@@ -0,0 +1,1178 @@
+bridge_address            = 16HEXDIG                                  ; bridge id
+regional_root_address     = 16HEXDIG                                  ; regional root bridge id
+root_address              = 16HEXDIG                                  ; root bridge id
+root_path_cost            = 1*9DIGIT                                  ; root path cost
+regional_root_cost        = 1*9DIGIT                                  ; regional root path cost
+root_max_age              = 1*2DIGIT                                  ; Max age of root bridge (6 to 40 sec, DEF:20sec)
+root_hello_time           = 1*2DIGIT                                  ; Hello time of root bridge (1 to 10 sec, DEF:2sec)
+root_forward_delay        = 1*2DIGIT                                  ; forward delay of root bridge (4 to 30sec , DEF:15sec)
+remining_hops             = 1*2DIGIT                                  ; Remaining Max-hops
+root_port                 = ifName                                    ; Root port name
+```
+
+#### STP_MST_PORT_TABLE
+```
+;Stores STP MST instance interface details
+key                 = _STP_MST_PORT_TABLE:"MST":id:ifname    ; MSTI+Intf with prefix "STP_MST_PORT
+port_number         = 1*3DIGIT                               ; port number or bridge port 
+path_cost           = 1*9DIGIT                               ; port path cost (1 to 200000000)
+priority            = 3*DIGIT                                ; port priority (0 to 240, DEF:128)
+port_state          = "state"                                ; DISABLED/DISCARDED/LISTENING/LEARNING/FORWARDING
+role                = "role"                                 ; DESIGNATED/ROOT/ALTERNATIVE/MASTER 
+desig_cost          = 1*9DIGIT                               ; designated cost
+external_cost       = 1*9DIGIT                               ; designated cost
+desig_root          = 16HEXDIG                               ; designated root
+desig_reg_cost      = 16HEXDIG                               ; designated root
+desig_bridge  	    = 16HEXDIG                               ; designated bridge
+desig_port    	    = 1*9DIGIT                               ; designated port
+fwd_transitions     = 1*5DIGIT                               ; number of forward transitions
+bpdu_sent           = 1*10DIGIT                              ; bpdu transmitted
+bpdu_received       = 1*10DIGIT                              ; bpdu received
+```
+
+#### STP_PORT_TABLE
+```
+;Stores STP interface detail
+key                 = _STP_PORT_TABLE:ifname    ; ifname with prefix STP_INTF
+edge_port           = BIT                       ; edge port  enabled or disabled
+link_type           = "type"                    ; point-to-point or shared link type
+mst_boundary        = BIT                       ; enabled or disabled
+mst_boundary_proto  = BIT                       ; enabled or disabled
+```
+
+#### STP_INST_PORT_FLUSH_TABLE 
```

Comment:
Why is STP_INST_PORT_FLUSH_TABLE needed in APP_DB? >> it is an action, shouldn't it be put on the APP DB or not?

Reply:
In general, Redis-DB is used for communication between dockers in the SONiC architecture. 
Are you looking of any alternative approaches to inter-docker communication in SONiC that you're familiar with?